### PR TITLE
Managing shadow of bottom navigation bar

### PR DESCRIPTION
**Pull request by heysadboy**
Solving issue https://github.com/fossasia/phimpme-android/issues/278

Now there is no space between view, as shadow is over it.

screenshot_2017-04-18-22-43-09
screenshot_2017-04-18-22-43-17

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @heysadboy we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/heysadboy/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by Abhi2424shek** [comment]
I can see a slight curve in the bottom navigation bar, can you see what is it please.

**Comment by heysadboy** [comment]
@Abhi2424shek I fixed that.

File: gradle.properties [hunk 1]
```diff
@@ -23,4 +23,4 @@ org.gradle.jvmargs=-XX:MaxPermSize=1024m -XX:+CMSClassUnloadingEnabled -XX:+Heap
 org.gradle.parallel=true
 # When set to true the Gradle daemon is used to run the build. For local developer builds this is our favorite property.
 # The developer environment is optimized for speed and feedback so we nearly always run Gradle jobs with the daemon.
-org.gradle.daemon=true
+org.gradle.daemon=true
```
Comment (Abhi2424shek) [inline gradle.properties]: Please don't remove a newline at EOF

**Comment by heysadboy** [comment]
okay @Abhi2424shek 👍

**Comment by CloudyPadmal** [comment]
Hello all @Abhi2424shek @heysadboy ! I guess this relates to this! :D my bottom navigation bar looks like this! Is it okay?

device-2017-04-23-061441

**Comment by heysadboy** [comment]
@CloudyPadmal It looks same here as earlier screenshot. Maybe it is due device. Can anyone else see how does this look like on their device? I think you also had issues earlier due to that.

**Comment by CloudyPadmal** [comment]
Yes @heysadboy it's seems related to my device. I'll check it from my side a little and tell :)

**Comment by Abhi2424shek** [comment]
It was looking fine when I tested.

**Comment by CloudyPadmal** [comment]
Yea it seems normal in other phones and even the code seems legit!! Have to figure out why strange in that phone :see_no_evil:

**Pull request merged**